packages/p/pinta/package.yml

```diff
@@ -1,32 +1,31 @@
 name       : pinta
-version    : 1.7.1
-homepage   : https://pinta-project.com/pintaproject/pinta/
-release    : 13
+version    : 2.1.1
+homepage   : https://www.pinta-project.com/
+release    : 14
 source     :
-    - https://github.com/PintaProject/Pinta/releases/download/1.7.1/pinta-1.7.1.tar.gz : cdbbbfe241b8fe5f3a1d0b16e73115125e264a9c94d25fce9e2fcf43429efab9
+    - git|https://github.com/PintaProject/Pinta.git : 2.1.1
 license    : MIT
 component  : multimedia.graphics
 summary    : Pinta Painting Made Simple
 description: |
     Pinta is a free, open source drawing/editing program modeled after Paint.NET. Its goal is to provide users with a simple yet powerful way to draw and manipulate images on Linux, Mac, Windows, and *BSD.
+libsplit   : no
+networking : yes
 builddeps  :
-    - pkgconfig(gtk-sharp-2.0)
-    - pkgconfig(mono)
-    - pkgconfig(mono-addins)
-    - pkgconfig(yelp-xsl)
-    - itstool
+    - pkgconfig(gtk+-3.0)
+    - autoconf-archive
+    - dotnet-sdk
+    - gettext
     - intltool
-    - yelp-devel
-    - yelp-tools
-rundeps    :
-    - mono
-    - mono-addins
-    - gtk-sharp
-patterns   :
-    - /usr/lib64/pkgconfig
 setup      : |
-    %configure
+    %patch -p1 -i $pkgfiles/0001-use-net8.patch
+    %autogen
+    sed -i '/^PINTA_BUILD_OPTS/ s/$/ -maxcpucount:1/' Makefile.in
 build      : |
-    %make -j1
-install    : |
-    %make_install -j1
+    %make
+install      : |
+    %make_install
+check      : |
+    %make check
+patterns   :
```

Comment:
Probably would be fine with just `libsplit: no`

Reply:
Without this pattern the /usr/lib64/pkgconfig/pinta.pc gets split out to a dev package. Not sure why that's t he case